documentation-site/components/walkthrough.js

```diff
@@ -0,0 +1,210 @@
+/*
+Copyright (c) 2018-2019 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+
+// @flow
+import * as React from 'react';
+import {
+  Modal,
+  ModalHeader,
+  ModalBody,
+  ModalFooter,
+  ModalButton,
+  SIZE,
+  ROLE,
+} from 'baseui/modal';
+import {StatefulTabs, Tab} from 'baseui/tabs';
+import {withStyle, useStyletron} from 'baseui';
+import {Button} from 'baseui/button';
+
+import {Label3, Label4, Paragraph3} from 'baseui/typography';
+import {StatefulTooltip} from 'baseui/tooltip';
+
+import CookiesConstructor from 'universal-cookie';
+const Cookies = new CookiesConstructor();
+
+// always change this if a new annoucement is rolled out!
+const CAMPAIGN_NAME = 'react-view';
+
+const COOKIE_NAME = `baseweb-news-${CAMPAIGN_NAME}`;
+
+export default () => {
+  const cookie = Cookies.get(COOKIE_NAME);
+  const [useCss, theme] = useStyletron();
+  const [isOpen, setIsOpen] = React.useState(!cookie);
+  const [currentPage, setPage] = React.useState(0);
+  const now = new Date();
+  Cookies.set(COOKIE_NAME, now.getTime(), {
+    // by default, cookies are session cookies, so without an expiration they are removed
+    // when the browser window is closed.
+    expires: new Date(now.getFullYear() + 10, now.getMonth()),
+    path: '/',
+  });
```

Comment:
This should be inside of the `React.useEffect` so it's not called on every single re-render?

Reply:
the thinking here was to keep the code simpler - this is the only code that runs: https://github.com/reactivestack/cookies/blob/master/packages/universal-cookie/src/Cookies.ts#L64-L76

should I still put this into a `useEffect`?

Reply:
I'll just quickly do this change

Reply:
@tajo should be ready for another pass